main.tf

```diff
@@ -2762,6 +2773,102 @@ resource "aws_cloudwatch_log_group" "io_bridge_cloudwatch_log_group" {
   tags = var.tags
 }
 
+// Terraformer
```

Comment:
Should this entire block be conditional on local.has_global_terraform_settings?

Reply:
AS commented above, another way to retrieve terraform info is from tags. So the lambda will be always installed no matter the customer uses the global settings or not.